yarn-project/acir-simulator/src/acvm/acvm.ts

```diff
@@ -23,6 +23,7 @@ export const ONE_ACVM_FIELD: ACVMField = `0x${'00'.repeat(Fr.SIZE_IN_BYTES - 1)}
 type ORACLE_NAMES =
   | 'packArguments'
   | 'getSecretKey'
+  | 'getNote'
```

Comment:
Leila removed this oracle in [her PR](https://github.com/AztecProtocol/aztec-packages/pull/1084) which is already on master. Just letting you know to avoid confusion.